service_worker.js

```diff
@@ -19,6 +19,8 @@ const CLS_THRESHOLD = 0.1;
 const FCP_THRESHOLD = 1800;
 const TTFB_THRESHOLD = 800;
 
+let optionsNoBadgeAnimation = false;
```

Comment:
doesn't the initial value of this need to be read from `chrome.storage.sync`?

Reply:
Good spot! Fixed in https://github.com/GoogleChrome/web-vitals-extension/pull/141/commits/010d88bc15b6f8ff6e048abb51bbf00d74f94fcd